doc/modules/model_evaluation.rst

```diff
@@ -215,6 +215,23 @@ the following two rules:
   Again, by convention higher numbers are better, so if your scorer
   returns loss, that value should be negated.
 
+In case you require the use of parallelisation (using ``n_jobs`` > 1) with a 
+custom scoring function, please make sure to define the custom function in a 
+module and import it for usage. Doing this, will ensure that the custom
```

Comment:
I would suggest : "While defining the custom scoring function in the original script should work out of the box with the default joblib backend (loky), importing it from another module will be more robust and work independently of the joblib backend".

Reply:
Would this be okay ?
"While defining the custom scoring function alongside the calling function should work
out of the box with the default joblib backend (loky), importing it from
another module will be a more robust approach and work independently of the joblib backend."

Reply:
> "While defining the custom scoring function alongside the calling function should work
> out of the box with the default joblib backend (loky), ~while~ importing it from
> another module will be a more robust approach and work independently of the joblib backend."

yes